shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/rule/builder/ShardingSphereRulesBuilder.java

```diff
@@ -103,13 +104,15 @@ private static void appendDefaultKernelSchemaRuleConfigurationBuilder(final Map<
     }
     
     @SuppressWarnings("rawtypes")
-    private static void appendDefaultKernelGlobalRuleConfigurationBuilder(final Map<RuleConfiguration, GlobalRuleBuilder> builders) {
+    private static Map<RuleConfiguration, GlobalRuleBuilder> appendDefaultKernelGlobalRuleConfigurationBuilder(final Map<RuleConfiguration, GlobalRuleBuilder> builders) {
```

Comment:
Is it necessary to do modification here?

Reply:
In my local as well as the process of ci, I found that produce abnormal ConcurrentModificationException, see the code found here has a todo, and multi-threaded read or add a builder, Can lead to a ConcurrentModificationException, so I just dropped in to repair the him.